frontend/src/App.tsx

```diff
@@ -1,27 +1,31 @@
-import { Route, BrowserRouter as Router, Routes, Navigate } from "react-router-dom";
+import {BrowserRouter as Router, Navigate, Route, Routes} from "react-router-dom";
 import Header from "./components/Header";
 import Footer from "./components/Footer";
 import StocksList from "./components/StocksList";
-import StockDetails from "./components/StockDetails";
 import "./App.css";
 import Home from "./pages/home/Home.tsx";
+import StockDetailsWithItems from "./components/StockDetailsWithItems.tsx";
 
 
 function App() {
-  return (
-    <Router>
-      <Header />
-      <main>
-        <Routes>
-            <Route path="/" element={<Navigate to="/home" />} />
-            <Route path="/home" element={<Home />} />
-            <Route path="/stocks" element={<StocksList />} />
-            <Route path="/stocks/:ID" element={<StockDetails />} />
-        </Routes>
-      </main>
-      <Footer />
-    </Router>
-  );
+    //const {ID} = useParams<{ ID: string }>();
+    return (
+        <Router>
+            <Header/>
+            <main>
+                <Routes>
+                    <Route path="/" element={<Navigate to="/home"/>}/>
+                    <Route path="/home" element={<Home/>}/>
+                    <Route path="/stocks" element={<StocksList/>}/>
+                    <Route path="/stocks/:ID" element={<StockDetailsWithItems/>}/>
+                    //TODO: check the need/relevance of these routes
+                    {/*<Route path="/stocks/:ID" element={<StockDetails />} />*/}
+                    {/*<Route path="/stocks/:ID/items" element={<StockItems  ID={ID}/>} />*/}
```

Comment:
issue(blocking): commented code to remove.

Reply:
cfr TODO